### DISCUSSION TITLE
Repository activity

### DISCUSSION THREAD

**Discussion by imakecodes** [butla/mountepy#14] [Q&A] [answered]
This repository is still active?
Since 2019 we don't have any new commit

**Comment by butla** [comment, marked as answer]
Hey. It's still active and the code is being used. It's pretty much "feature-complete" when it comes to talking with Mountebank.

There's a bunch of things I'd like to do with the code:
- replace Mountebank with ExternalMountebank, since running Mountebank in Docker is probably the default (don't know if anybody is using this `mb` process runner I made - I certainly don't anymore).
- add CI again
- add a feature where you can override an existing impostor (delete, stash remembered requests, apply a new one, restore old one afterwards)

They just never managed to get to the top of my priority list :)

**Reply by imakecodes** [reply]
Great news!!! 

Thank you for sharing this
